test/wait/contrail.go

```diff
@@ -93,6 +93,32 @@ func (c Contrail) ForPostgresActive(name string) error {
 	return err
 }
 
+// ForPodImageChange is used to wait until Image is updated
+func (c Contrail) ForPodImageChange(kubeClient kubernetes.Interface, name string, newImage string, container string) error {
```

Comment:
I would rename it to `WaitForPodImageChange`

Reply:
Please take a look at how this function is being used/called. That should convince you about the name.